docs/readthedocs/source/doc/Nano/Overview/nano.md

```diff
@@ -45,11 +45,11 @@ In a pure pip environment, you need to run `source bigdl-nano-init` every time y
 
 #### **3.1 PyTorch**
 
-BigDL-Nano supports both PyTorch and PyTorch Lightning models and most optimizations requires only changing a few "import" lines in your code and adding a few flags.
+BigDL-Nano supports both PyTorch and PyTorch Lightning models and most optimizations require only changing a few "import" lines in your code and adding a few flags.
 
 BigDL-Nano uses a extended version of PyTorch Lightning trainer for integrating our optimizations.
 
-For example, if you are using a LightingModule, you can use the following code to enable intel-extension-for-pytorch and multi-instance training.
+For example, if you are using a LightningModule, you can use the following codes to enable intel-extension-for-pytorch and multi-instance training.
```

Comment:
I think code is not countable when used in the context of programming. Maybe "the following lines code" or the "the following code snippet"

Reply:
Have fixed it :D